resources/automate.rb

```diff
@@ -21,6 +21,8 @@
 provides :chef_automate
 resource_name :chef_automate
 
+unified_mode true
```

Comment:
It woudl be great to use the if responds on this call so we can maintain compatibility with older chef releases. We use this internally in some projects where we haven't gotten them off 15 yet.